docs/demos/coast-to-coast-q4-2023.md

```diff
@@ -0,0 +1,793 @@
+# Coast to Coast Demo for Q4 2023
+
+This coast to coast demo highlights some of the new features introduced to OLMv1 in Q4 of 2023.
+
+New Features:
+- `Catalog` polling
+- Version range support on the `spec.version` field for `Operator` resources
+- Semver upgrade constraint policy
+- `BundleDeployment` health status
+
+This document will go through an entire coast to coast example that highlights each of these new features. Each step will be documented as if you are following along and will be done sequentially. This document will also use some values specific to this example and in your own projects should be replaced to use a different value.
+
+## Prerequisites
+The following prerequisites are required:
+- A running Kubernetes cluster where you have admin priveleges
+- `operator-sdk` installed. Installation instructions can be found at https://sdk.operatorframework.io/docs/installation/
+- `kubectl` installed. Installation instructions can be found at https://kubernetes.io/docs/tasks/tools/
+- a container runtime of your choice. In this demo we will be using `docker`
+- Install `operator-controller` vX.Y.Z. Installation instructions can be found at https://github.com/operator-framework/operator-controller/releases/tag/vX.Y.Z
+- `kustomize` installed. Installation instructions can be found at https://kubectl.docs.kubernetes.io/installation/kustomize/
+
+## Prepare for takeoff
+
+Before we can start exploring the new features we need to do some preparation by creating a new
+operator and building a few different versions.
```

Comment:
Perhaps a downstream bias, but I default to always capitalizing _Operator_, unless it is referring to specific object and is in back ticks.

Reply:
[08e8dbb](https://github.com/operator-framework/operator-controller/pull/533/commits/08e8dbba77679d610d04f2b8002613fe94421aff)